.gitignore

```diff
@@ -3,6 +3,7 @@
 /.release
 /.tarballs
 debug/
+.idea/
```

Comment:
Please exclude the JetBrains `.idea` from the pull request. This should probably be added to the `.gitignore`.

Reply:
I don't know what I was thinking. `.idea/` is in `.gitignore.